controllers/repliesController/getAllReplies.js

```diff
@@ -31,9 +32,19 @@ const getAllReplies = async (req, res, next) => {
       return next(new CustomError(404, " Comment not found "));
     }
 
-    const replies = await Replies.find({ commentId: commentId }).populate(
-      "replyOwner"
-    );
+    // Create query for replies.
+    let query = {};
+
+    query.commentId = commentId;
+    if (ownerId) query.ownerId = ownerId;
+
+    if (!!isFlagged === true) {
```

Comment:
you might be able to change this to:

```javascript
if (isFlagged) query["flags.0"] = {"$exists": true};
```

Reply:
Let me try that. Will get back to you with the outcome.

Reply:
tested working on gitpod 
![image](https://user-images.githubusercontent.com/17856665/86789547-5644d880-c05f-11ea-98af-71f996a3c812.png)